handlers/serviceset.go

```diff
@@ -167,6 +176,22 @@ func (s *ServiceSet) logError(format string, args ...interface{}) {
 	}
 }
 
+// reloadEndpointHandler
+func (s *ServiceSet) reloadEndpointHandler(w http.ResponseWriter, r *http.Request) {
+	token := r.URL.Query().Get("token")
+	if s.enableReloadEndpoint && token == s.reloadToken {
```

Comment:
i think it's more go-esque to check for err and return early

Reply:
sorry, i dont quite understand. where would i check for an err here?

Reply:
eh sorry, i misread something. good to go.